.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,19 +4,38 @@ about: Report a reproducible or regression bug in React Native Testing Library'
 labels: 'bug report'
 ---
 
-## Versions
-
-<!-- Put versions of react, react-native, react-native-testing-library and react-test-renderer you are using -->
-
-## Description
+## Describe the bug
 
 <!--
   Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know.
 -->
 
-## Reproducible Demo
+## Expected behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Steps to Reproduce
 
 <!--
   Let us know how to reproduce the issue. Include a code sample or share a project that reproduces the issue.
   Please follow the guidelines for providing a minimal example: https://stackoverflow.com/help/mcve.
 -->
+
+Please provide detailed steps for reproducing the issue.
+1. Navigate to...
+2. Press on...
+3. Scroll to...
+4. See error..
```

Comment:
Nah, I'd like to avoid it. Without repository with repro, there's a 100% chance I'm not gonna try to reproduce it myself.

Reply:
ok, will remove this part